src/helpers/xml-builder.js

```diff
@@ -1450,25 +1465,30 @@ const buildParagraph = async (vNode, attributes, docxDocumentInstance) => {
 
           computeImageDimensions(childVNode, modifiedAttributes);
         }
-        const runOrHyperlinkFragments = await buildRunOrHyperLink(
-          childVNode,
-          isVNode(childVNode) && childVNode.tagName === 'img'
-            ? { ...modifiedAttributes, type: 'picture', description: childVNode.properties.alt }
-            : modifiedAttributes,
-          docxDocumentInstance
-        );
-        if (Array.isArray(runOrHyperlinkFragments)) {
-          for (
-            let iteratorIndex = 0;
-            iteratorIndex < runOrHyperlinkFragments.length;
-            iteratorIndex++
-          ) {
-            const runOrHyperlinkFragment = runOrHyperlinkFragments[iteratorIndex];
-
-            paragraphFragment.import(runOrHyperlinkFragment);
-          }
+        if (childVNode.tagName === 'hr') {
```

Comment:
note to self- test this in a few permutations

Reply:
(Writing this second comment after seeing the other code on the bottom adding in the buildHorizontalRule)

Add test cases that distinguish between these two scenarios where we call `buildHorizontalRule()`